pkg/symbol/addr2line/symtab.go

```diff
@@ -106,14 +109,54 @@ func (lnr *SymtabLiner) PCToLines(addr uint64) (lines []profile.LocationLine, er
 // to facilitate searching.
 func symtab(objFile *elf.File) ([]elf.Symbol, error) {
 	syms, sErr := objFile.Symbols()
+	dynSyms, dErr := objFile.DynamicSymbols()
+
+	var pltSymbols []elf.Symbol
+	// see symbol-elf.c/dso__synthesize_plt_symbols
+	if pltRelSection, pltSection := objFile.Section(".rela.plt"), objFile.Section(".plt"); dErr == nil &&
```

Comment:
🔧  🏕 Could we please break down this condition into maybe predictor functions? It is tough to read.

Reply:
haha, you can edit the code directly~.

I am not familiar with code style.
It is hard for me to know what to do.
It is a great way for me to learn it.